nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/reporting/ReportLineageToAtlas.java

```diff
@@ -91,10 +91,12 @@
 import static org.apache.nifi.reporting.util.provenance.ProvenanceEventConsumer.PROVENANCE_START_POSITION;
 
 @Tags({"atlas", "lineage"})
-@CapabilityDescription("Publishes NiFi flow data set level lineage to Apache Atlas." +
-        " By reporting flow information to Atlas, an end-to-end Process and DataSet lineage such as across NiFi environments and other systems" +
-        " connected by technologies, for example NiFi Site-to-Site, Kafka topic or Hive tables." +
-        " There are limitations and required configurations for both NiFi and Atlas. See 'Additional Details' for further description.")
+@CapabilityDescription("Report NiFi flow data set level lineage to Apache Atlas." +
+        " End-to-end lineages across NiFi environments and other systems can be reported those are" +
```

Comment:
> can be reported those are

Did you mean something like *can be reported **if** those are*

Reply:
Yes, I meant something like if/when/even if ... I added 'if' to make it sounds more natural. Thank you, @MikeThomsen !